### DISCUSSION TITLE
關於作業繳交所需之log, report檔案

### DISCUSSION THREAD

**Discussion by gene5487** [bol-edu/HLS-SOC-Discussions#106] [Q&A]
大家好，想請問作業作業繳交所需之log, report檔案 (如下圖所列)，使用Vivado應如何產生? 謝謝。 

![img](https://github.com/bol-edu/HLS-SOC-Discussions/assets/58682521/e75f47a4-2ff1-4bfc-9196-68892d5dd031)

**Comment by ZheChen-Bill** [comment]
simulation的log檔在simluation的路徑下(應為project/project.sim/behavior/xsim)，synthesis report也可以在runs/synth裡面找到，關於timing report可以參考SYN_Workflow

**Reply by hbx1241** [reply]
不好意思，請問有SYN_Workflow的連結嗎？

**Reply by ZheChen-Bill** [reply]
之前已經有在lab-fir裡面更新上去了，可以去lab-fir的github頁面確認一下。

**Reply by kevin861222** [reply]
[SYN_Workflow.pptx](https://github.com/bol-edu/HLS-SOC-Discussions/files/13061068/SYN_Workflow.pptx)
